nf_core/create.py

```diff
@@ -455,34 +439,24 @@ def fix_linting(self):
         # Add the lint content to the preexisting nf-core config
         nf_core_yml = nf_core.utils.load_tools_config(self.outdir)
         nf_core_yml["lint"] = lint_config
-        with open(os.path.join(self.outdir, ".nf-core.yml"), "w") as fh:
+        with open(self.outdir / ".nf-core.yml", "w") as fh:
             yaml.dump(nf_core_yml, fh, default_flow_style=False, sort_keys=False)
 
-        # The dumped yaml file will not follow prettier formatting rules
-        # so we run prettier on the file
-        try:
-            subprocess.run(
-                ["prettier", "--write", os.path.join(self.outdir, ".nf-core.yml")],
-                stdout=subprocess.DEVNULL,
-                stderr=subprocess.DEVNULL,
-                check=False,
-            )
-        except FileNotFoundError:
-            log.warning(
-                "Prettier is not installed. Please install it and run it on the pipeline to fix linting issues."
-            )
+        run_prettier_on_file(os.path.join(self.outdir, ".nf-core.yml"))
 
     def make_pipeline_logo(self):
         """Fetch a logo for the new pipeline from the nf-core website"""
 
         logo_url = f"https://nf-co.re/logo/{self.template_params['short_name']}?theme=light"
         log.debug(f"Fetching logo from {logo_url}")
 
-        email_logo_path = f"{self.outdir}/assets/{self.template_params['name_noslash']}_logo_light.png"
+        email_logo_path = self.outdir / "assets" / f"{self.template_params['name_noslash']}_logo_light.png"
         self.download_pipeline_logo(f"{logo_url}&w=400", email_logo_path)
         for theme in ["dark", "light"]:
             readme_logo_url = f"{logo_url}?w=600&theme={theme}"
-            readme_logo_path = f"{self.outdir}/docs/images/{self.template_params['name_noslash']}_logo_{theme}.png"
+            readme_logo_path = (
```

Comment:
I have to say, the old version is easier to read for me.

Reply:
I had a similar feeling when I changed it - nevertheless, I thought getting the path separators out of the string was a win. Another alternative would have been to wrap the long f-sting with Path(). Also, I would love to see the pathlib way of path handling be used consistently throughout the codebase. Please let me know if you (still) think I should revert this change.